docs/source/transforms.rst

```diff
@@ -162,9 +192,15 @@ Conversion
     :template: class.rst
 
     ToPILImage
+    v2.ToPILImage
```

Comment:
We only kept this for BC but prefer `ToImagePIL` since it aligns better with our naming scheme. Should we keep it out of the documentation?

Reply:
> From a user perspective, I don't think "I should be updating my code to remove ToPILImage and use ToImagePIL instead because the torchvision maintainers have a new naming scheme" is a super compelling story.

Agreed. This is why we left it in as alias:

https://github.com/pytorch/vision/blob/928b05cad36eadb13e169f03028767c8bcd1f21d/torchvision/transforms/v2/_type_conversion.py#L72-L74

Meaning, only users who don't have old pipelines are going to see this and there it feels ok to not show the old transform.

In case you still want to, can we maybe just let the docstring say "alias of X" like PyTorch core is doing in such situations? For example:

https://pytorch.org/docs/stable/generated/torch.not_equal.html?highlight=not_equal#torch.not_equal

Reply:
yeah, this is a sphinx built-in behaviour I believe, and this is what we already have:

![image](https://user-images.githubusercontent.com/1190450/220408025-9bfa818f-254e-43ca-8f89-477bbd4b4de9.png)


![image](https://user-images.githubusercontent.com/1190450/220408060-d7b72e2b-8910-4af3-ad1b-e11f77fdc1f5.png)